templates/repo/issue/view_content/comments.tmpl

```diff
@@ -318,7 +318,11 @@
 		</div>
 	{{else if eq .Type 22}}
 		<div class="event" id="{{.HashTag}}">
-			<span class="issue-symbol">{{svg (printf "octicon-%s" .Review.Type.Icon) 16}}</span>
+			{{svg "octicon-primitive-dot" 16}}
+			<span class="issue-symbol {{if eq .Review.Type 1}}green
+			{{else if eq .Review.Type 2}}grey
+			{{else if eq .Review.Type 3}}red
+			{{else}}grey{{end}}">{{svg (printf "octicon-%s" .Review.Type.Icon) 16}}</span>
```

Comment:
Would just put this on a single line to avoid newlines in the output.

Reply:
Alternatively, go templates allow [the following syntax](https://golang.org/pkg/text/template/#hdr-Text_and_spaces):

```
			<span class="issue-symbol {{if eq .Review.Type 1}}green
			{{- else if eq .Review.Type 2}}grey
			{{- else if eq .Review.Type 3}}red
			{{- else}}grey{{end}}">{{svg (printf "octicon-%s" .Review.Type.Icon) 16}}</span>
```

(hint: check the `- ` preceeding the `else`)

Reply:
sorry , I just copy it from othe place. you are right , I will change it